module_utils/network_lsr/argument_validator.py

```diff
@@ -2012,3 +2083,101 @@ def _ipv6_is_not_configured(connection):
                             "match.path is not supported by the running version of "
                             "NetworkManger.",
                         )
+        self.validate_route_tables(connection, idx)
+
+
+class IPRouteUtils(object):
+
+    ROUTE_TABLE_ALIAS_RE = re.compile("^[a-zA-Z0-9_.-]+$")
+
+    @classmethod
+    def _parse_route_tables_mapping(cls, file_content, mapping):
+        for line in file_content.split(b"\n"):
+
+            #
+            # We are a bit more forgiving, but the aliases must be nice UTF-8 names.
+
+            # iproute2 only skips over leading ' ' and '\t'. We are more forgiving,
+            # and strip all leading+trailing whitespace.
+            line = line.strip()
+
+            # skip empty lines or comments
+            if not line:
+                continue
+            if line[0] == ord(b"#"):
+                continue
```

Comment:
You can simplify this on a single if:

```
if not line or line[0] == ord(b"#"):
    continue
```
The `or` operator is lazy, so if `line` is present it won't evaluate the next condition `line[0] == ord(b"#")`.

Reply:
The reason that I separated the condition is because I want to make it more clear. (skip empty lines or comments during parsing) But either way is fine.